pkg/resources/cloudcontroller/kubevirt.go

```diff
@@ -90,8 +90,8 @@ func kubevirtDeploymentCreator(data *resources.TemplateData) reconciling.NamedDe
 									},
 								},
 								{
-									ConfigMap: &corev1.ConfigMapProjection{
-										LocalObjectReference: corev1.LocalObjectReference{Name: resources.CloudConfigConfigMapName},
+									Secret: &corev1.SecretProjection{
```

Comment:
KubeVirt cloud config doesn't have to be a secret, it doesn't contain any credentials. However, if you would need to change a lot of things only because of that I think it's acceptable that it's mounted as a secret.

Reply:
Just for consistency reasons I would simply always use a Secret. Are there any downsides, except that it might seem overblown?

Reply:
There are no other downsides.